Bio/_py3k/__init__.py

```diff
@@ -135,6 +135,14 @@ def _binary_to_string_handle(handle):
     from urllib.parse import urlencode, quote
     from urllib.error import URLError, HTTPError
 
+    exec("""\
```

Comment:
Is the use of ``exec`` to avoid a syntax error on Python 2?

Reply:
Yes